code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm

```diff
@@ -2,7 +2,7 @@
 //They're generic weapons for the refinery, which is fine
 /obj/item/ego_weapon/city/streetlight_greatsword
 	name = "streetlight greatsword"
-	desc = "A greatsword used by the streelight office."
+	desc = "A greatsword used by a fixer with big shoes to fill.'Maybe... I should have told her how I feel.'"
```

Comment:
How I felt*

Reply:
Could still feel it, it works either way

Reply:
"have told her" is past tense. While I'd normally say an argument over semantics is pointless, writing quality is the entire point of this PR

Reply:
That particular line is a direct quote from Library of Ruina, but I see your point! I'll make a note to go back over it in my next batch.